.infrastructure/50_storage.tf

```diff
@@ -23,6 +23,26 @@ resource "aws_s3_bucket_versioning" "cms_medialibrary_bucket" {
   }
 }
 
+data "aws_iam_policy_document" "cms_iam_policy" {
+  statement {
+    actions = ["s3:GetObject", "s3:ListBucket"]
+    resources = [
+      "${aws_s3_bucket.cms_medialibrary_bucket.arn}",
+      "${aws_s3_bucket.cms_medialibrary_bucket.arn}/*"
+    ]
+
+    principals {
+      type        = "AWS"
+      identifiers = [aws_cloudfront_origin_access_identity.main.iam_arn]
+    }
+  }
+}
+
+resource "aws_s3_bucket_policy" "cloudfront_cms" {
+  bucket = aws_s3_bucket.cms_medialibrary_bucket.id
+  policy = data.aws_iam_policy_document.cms_iam_policy.json
+}
+
```

Comment:
What about move them into [.infrastructure/09_iam_role.tf](https://github.com/pagopa/b2b-portals/blob/2e29d0f/.infrastructure/09_iam_role.tf) file?

Reply:
not being a change to a role but an addition of policy to the S3 resource and to maintain consistency with the same privilege assigned to the other S3 resource (see rows 71-89 for S3 "website"), i have inserted it into this file. But if we want we can move it to the "09_iam_role .tf" file

Reply:
Right! 👍